website/static/js/code-block-buttons.js

```diff
@@ -0,0 +1,47 @@
+// Turn off ESLint for this file because it's sent down to users as-is.
+/* eslint-disable */
```

Comment:
Please try to fix the problems instead of disable here.

Reply:
Sure, I've updated the PR.